userland/libtock/ble.c

```diff
@@ -1,43 +1,123 @@
+/*
+ * BLE setup functions
+ */
+
 #include "ble.h"
+#include "tock.h"
+#include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <string.h>
 
-int ble_adv_set_txpower(BLE_TX_Power_t power) {
-  return command(DRIVER_RADIO, BLE_CFG_TX_POWER, power);
+#define GAP_DATA_MAX_SIZE 31
+
+
+
+// helper functions
+static int s_ble_cfg_addr (uint8_t *addr, uint8_t addr_len);
+static int s_ble_cfg_flags (uint8_t flags);
+static int s_ble_cfg_adv_itv (uint16_t advertising_itv_ms);
+static int s_ble_cfg_gap_data (GapAdvertisementData_t header, uint8_t *data, uint8_t len);
+
+// local VARIABLES
+static uint8_t s_gap[GAP_DATA_MAX_SIZE];
```

Comment:
What does the `s` stand for?

Reply:
static, well I named all things that have file scope as s_xxx_yyy but I can remove it if it's annoying